src/frontend/sharedTypes/navigation.ts

```diff
@@ -168,12 +168,13 @@ export type RootStackParamsList = {
   ProjectStatsTurnedOff: undefined;
   EarlyAccess: undefined;
   EarlyAccessOff: undefined;
-  LeaveProject: undefined;
+  LeaveProject: {memberType: 'coordinator' | 'participant'};
   LeftProjectConfirmation: undefined;
   CollaboratorInfo: {
     deviceId: string;
     isOwnDevice: boolean;
     deviceType: DeviceType;
```

Comment:
i don't think we should be passing deviceType. In `CollaboratorInfo` we are using `useSingleMember` which has access to deviceType